code-pushup.config.ts

```diff
@@ -110,6 +115,30 @@ const config: CoreConfig = {
         },
       ],
     },
+    {
+      slug: 'security',
+      title: 'Security',
+      refs: [
+        {
+          type: 'group',
+          plugin: 'js-packages',
+          slug: 'npm-audit',
+          weight: 1,
+        },
+      ],
+    },
+    {
+      slug: 'dependencies',
+      title: 'Dependencies',
```

Comment:
Shouldn't...it be outdated dependencies instead of updated?

Reply:
We're scoring how up-to-date the dependencies are, not how outdated they are. Otherwise it's sounds like the inverse is good. 

Could also be "Up-to-date dependencies",  "Updates" or even "Modern tech stack". Something that makes it clear what the goal is.